SemanticAnalysis/parser.y

```diff
@@ -189,11 +215,19 @@ assign_stmt:                      param assignment
                                   {
                                     printf("assign_stmt\n");
                                     $$ = makeNode(astAssignStmt, NULL, $1, $2, NULL, NULL);
+                                    popV();
+                                    popV();
                                     // sym = find_variable(param->)
                                   }
                                   | arr assignment
                                   {
-                                    $$ = makeNode(astArrayAssignStmt, NULL, $1, $2, NULL, NULL);
+                                    $$ = makeNode(astAssignStmt, NULL, $1, $2, NULL, NULL);

```

Comment:
```c
$$ = makeNode(astAssignStmt, NULL, $1, $2, NULL, NULL);
```
Why did you change `astArrayAssignStmt` to `astAssignStmt`?

Reply:
I'll change this back.

Reply:
Resolved this.